src/sentry/models/project.py

```diff
@@ -106,6 +106,7 @@ class Project(Model, PendingDeletionMixin):
         flags=(
             ("has_releases", "This Project has sent release data"),
             ("has_sourcemaps", "This Project has processed source maps"),
+            ("has_issue_alerts_targeting", "This Project has issue alerts targeting"),
```

Comment:
Oh right, you'll need to generate a migration for this. It won't make any actual changes, but updates the model state in the migration tool.